### PR TITLE
Reorganize repo and minor data-fixes

### PR DESCRIPTION
**Pull request by eeyun**
Ok, this one ballooned a bit. I apologize, once we're in a clean state with this I'll do my best to keep my PRs more focused on single features/bugs.

This reorganizes the repo (which includes renaming somethings and creating some different directories. There should be _zero_ functional changes in this code. Everything should effectively be in the exact state it was before the merge but with some very minor bugfixes.

Primary changes:
- Move away from .mjs module model (this is most of the changes to the files in `module`.)
- Data model bugfix: Specifically the _max_ value for skill expertise and focus. That change is what triggered the changes to the html template files. While fixing those up, I also found that the actor entity instantiation for health as well as TN values was broken. I've corrected those issues while I was in there. To be clear - health is still incorrect as current resolve and max resolve are bound together. This can be fixed later.
- Data model bugfix:  Moved renown into background object
- Added myself to the authors list
- Deleted unused binary artifacts.

Signed-off-by: Ian Henry <ianjhenry00@gmail.com>

### CODE REVIEW

File: css/conan2d20.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import url("https://fonts.googleapis.com/css2?family=Martel:wght@400;800&family=Roboto:wght@300;400;500&display=swap");
 @font-face {
   font-family: 'celticbold';
-  src: url("/systems/conan2d20/ui/celtic-webfont.woff2") format("woff2"), url("/systems/conan2d20/ui/celtic-webfont.woff") format("woff"), url("/systems/conan2d20/ui/CELTIC.TTF") format("truetype");
+  src: url("/systems/conan2d20/fonts/CELTIC.TTF") format("truetype");
```
Comment (jeromelavoie) [inline css/conan2d20.css:4]: why remove all other formats of fonts ? is this best practice to have all formats so the browser can use it's pefered format ?

File: module/actor/entity.js [hunk 2]
```diff
@@ -42,29 +40,30 @@ export class Conan2D20Actor extends Actor {
...
-      skill.tn = skill.value + data.attributes[skill.attribute].value;
+      skill.tn = skill.expertise.value + data.attributes[skill.attribute].value;
     }
 
     // Prepare Upkeep Cost
     // 3 gp + [Standing] - [Renown]
-    data.resources.upkeep = 3 + data.background.standing.value - data.background.renown;
 
-    //console.log(data);
+    data.resources.upkeep.value = 3 + data.background.standing.value - data.background.renown;
```
Comment (jeromelavoie) [inline module/actor/entity.js:66]: i'm wondering about ressources vs background and that you might have had the correct idea about putting upkeep, standing and renown in ressources.

File: templates/actors/includes/actor-tabs-skills.html [hunk 3]
```diff
@@ -11,75 +11,75 @@
         <span class="skill-mod rollable" data-roll="2d20cs<={{data.skills.acr.tn}}" data-label="acr"><label
                 for="data.skills.acr.value" class="skill-label">{{ localize "conan2d20.skills.acr" }}</label></span>
         <div>
-            <input type="text" name="data.skills.acr.value" value="{{data.skills.acr.value}}" data-dtype="Number" />
-            <input type="text" name="data.skills.acr.focus" value="{{data.skills.acr.focus}}" data-dtype="Number" />
+            <input type="text" name="expertise" value="{{data.skills.acr.expertise.value}}" data-dtype="Number" />
+            <input type="text" name="focus" value="{{data.skills.acr.focus.value}}" data-dtype="Number" />
```
Comment (jeromelavoie) [inline templates/actors/includes/actor-tabs-skills.html:15]: My understanding was that the name field was told the system where to save the value when the form was submitted. hence why I put the full path there (changing the skill expertise and focus used to work before the refactor, don't know yet if it still works)

File: css/conan2d20.css (hunk 1 above)
Comment (eeyun) [reply css/conan2d20.css:4]: Actually this file just didn't exist in the repo and was causing an error in the server logs on startup.

File: module/actor/entity.js (hunk 2 above)
Comment (eeyun) [reply module/actor/entity.js:66]: I was thinking about this as well. I think for now its an easy migration if we want to change it. I think standing and renown _can_ change but they're not exactly a spendable resource like upkeep. FWIW its not a big deal to make comparisons across data objects like this :+1:

File: templates/actors/includes/actor-tabs-skills.html (hunk 3 above)
Comment (eeyun) [reply templates/actors/includes/actor-tabs-skills.html:15]: I think you are correct, I was thinking this was a standard html decorator. I'll fix that up. Ultimately i think we can write these as a loop to clean up our syntax but we can do that in a future pr! Great Catch!

**Pull request merged**